prompting/tools/datasets/wiki.py

```diff
@@ -177,7 +182,7 @@ def get(
         name: str,
         selector: Selector = None,
         include: List = None,
-        exclude: List = None,
+        exclude: List = [],
```

Comment:
`None `default type would be better for lists/dicts/etc to avoid shared state across calls

Reply:
Yeah you're totally right, this was an accidental keepsake from when I was trying a different approach